EpiAware/test/predictive_checking/fast_approx_for_r.jl

```diff
@@ -0,0 +1,73 @@
+#=
+# Fast approximation for `r` from `R₀`
```

Comment:
How are you rendering these quickly? I am having to using literate.jl -> rename to jmd -> weave.jl and its pretty dull

Reply:
I run as a script saving any figures as an asset and then render literate -> github flavoured markdown.

The reason for using literate is that when we go to Documenter it renders to documenter flavoured md which is advantageous.

If there wasn't an eventual Documenter target, I'd probably either literate -> jupyter OR use Pluto on local Pkg mode